api/v1alpha1/server_types.go

```diff
@@ -222,6 +222,13 @@ type ServerStatus struct {
 	// +optional
 	State ServerState `json:"state,omitempty"`
 
+	// NetworkCheck indicates the network configuration validation status.
+	// Possible values: "Pass" (all interfaces match Netbox), "Fail" (mismatches detected), "Unknown" (not yet validated).
+	// +kubebuilder:validation:Enum=Pass;Fail;Unknown
+	// +kubebuilder:default=Unknown
+	// +optional
+	NetworkCheck string `json:"networkCheck,omitempty"`
```

Comment:
Who exactly should react on changes to that field? Also this looks like something which is better kept in the `status.conditions`.